docs/integrations/lightgbm.rst

```diff
@@ -3,33 +3,74 @@
 Neptune-LightGBM Integration
 ============================
 
+|Run on Colab|
+
+What will you get with this integration?
+----------------------------------------
+
+|lightgbm-tour|
+
 |lightGBM| is a popular gradient boosting library. The integration with Neptune lets you log training and evaluation metrics and have them visualized in Neptune.
+   
+.. note::
 
-.. image:: ../_static/images/integrations/lightgbm_neptuneml.png
-   :target: ../_static/images/integrations/lightgbm_neptuneml.png
-   :alt: lightGBM neptune.ai integration
+    This integration is tested with ``neptune-client==0.4.129``, ``neptune-contrib==0.25.0``, and ``lightgbm==2.2.3``, but should also work with later versions of ``neptune-client`` and ``neptune-contrib``
+	
+.. _lightgbm-quickstart:
+
+Quickstart
+----------
+
+This quickstart will show you how to:
+
+* Install the necessary Neptune packages
+* Log lightGBM metrics to and visualize them with Neptune
 
-Requirements
-------------
-To use Neptune + lightGBM integration you need to have installed is |neptune-client| and |neptune-contrib|.
+|Run on Colab|
 
-.. code-block:: bash
+.. _lightgbm-before-you-start-basic:
 
-    pip install neptune-client neptune-contrib[monitoring]
+Before you start
+^^^^^^^^^^^^^^^^
 
-Initialize Neptune and create an experiment
--------------------------------------------
+#. Ensure that you have ``Python 3.x`` and following libraries installed:
+
+   * ``neptune-client>=0.4.129``. See |neptune-client|
+   * ``neptune-contrib>=0.25.0``. See |neptune-contrib|
+   * ``lightgbm==2.2.3``. See the |lightgbm-install|
+   
+   .. code-block:: bash
+   	
+      pip install --quiet lightgbm==2.2.3 neptune-client neptune-contrib[monitoring]
+
+#. You also need minimal familiarity with lightGBM. Have a look at the |lightgbm-guide| guide to get started.
+
+Step 1: Initialize Neptune
+^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 .. code-block:: python3
 
     import neptune
 
-    neptune.init(api_token='ANONYMOUS',
-                 project_qualified_name='shared/showroom')
-    neptune.create_experiment(name='lightGBM-training')
+    neptune.init(api_token='ANONYMOUS', project_qualified_name='shared/showroom')
```

Comment:
There is a shared project `shared/LightGBM-integration` so please log there. 
https://ui.neptune.ai/o/shared/org/LightGBM-integration/experiments?viewId=7397cbb5-5291-4d18-921b-04fe39ab23bf